src/webui/www/private/scripts/client.js

```diff
@@ -353,6 +408,60 @@ window.addEvent('load', function() {
         }
     };
 
+    const updateTagList = function() {
+        const tagFilterList = $('tagFilterList');
+        if (!tagFilterList)
```

Comment:
Can you make this check more explicit? Directly compare to undefined/null or whatever it would be in a failure case (I can't immediately tell, hence wanting the explicitness). Same with while loop below.